crates/shared/src/price_estimation/sanitized.rs

```diff
@@ -0,0 +1,245 @@
+use super::{ensure_token_supported, Estimate, PriceEstimating, PriceEstimationError, Query};
+use crate::bad_token::BadTokenDetecting;
+use anyhow::Result;
+use futures::future;
+use model::order::BUY_ETH_ADDRESS;
+use primitive_types::{H160, U256};
+use std::sync::Arc;
+
+const ETH_UNWRAP_COST: U256 = U256([0, 0, 0, 123]);
```

Comment:
Just wanted to let you know that we already have a module that we keep gas cost constants for this kind of thing (I think its `shared/src/price_estimation/gas.rs`). I slightly prefer having all these things in the same module. That being said, this gas cost is very specific to this price estimator, so I see the argument for keeping it here.

I don't feel strongly either way, just thought I'd point out the existence of this module in case you missed it.

Reply:
Done